lib/vva/services.rb

```diff
@@ -6,7 +6,7 @@
 module VVA
   class Services
     def initialize(wsdl:, username:, password:, log: false,
-                   ssl_cert_file:, ssl_cert_key_file:, ssl_ca_cert:)
+                   ssl_cert_file: nil, ssl_cert_key_file: nil, ssl_ca_cert: nil)
```

Comment:
Allow nil values as it turns out VVA doesn't require us to send our certificate. I don't know what happened with that.